src/sample_maps/2base_1room_1robot.json

```diff
@@ -0,0 +1,96 @@
+{
+    "room": {
+        "class": "CompoundPhysical",
+        "position": [0,0,0,0,0],
+        "subobjects": {
+            "wall0":{
+                "class": "Box",
+                "shape": {
+                    "class": "Cube",
+                    "arguments": {
+                        "length": 4,
+                        "height": 3,
+                        "width": 0.2
+                    }
+                },
+                "position": [0,4,0,0,90]
+            },
+            "wall1":{
+                "class": "Box",
+                "shape": {
+                    "class": "Cube",
+                    "arguments": {
+                        "length": 4,
+                        "height": 3,
+                        "width": 0.2
+                    }
+                },
+                "position": [0,-4,0,0,90]
+            },
+            "wall2":{
+                "class": "Box",
+                "shape": {
+                    "class": "Cube",
+                    "arguments": {
+                        "length": 4,
+                        "height": 3,
+                        "width": 0.2
+                    }
+                },
+                "position": [4,0,0,90,0]
+            },
+            "wall3":{
+                "class": "Box",
+                "shape": {
+                    "class": "Cube",
+                    "arguments": {
+                        "length": 4,
+                        "height": 3,
+                        "width": 0.2
+                    }
+                },
+                "position": [-4,0,0,90,0]
+            }
+        }
+    },
+    "robot1": {
+        "class": "VacuumCleanerV0",
+        "position": [0,0,0,45,90],
+        "shape": {
+            "class": "Cylinder",
+            "arguments": {
+                "radius": 0.2,
+                "height": 1
+            }
+        },
+        "bases": ["base-A","base-B"]
+    },
+    "base-A": {
```

Comment:
this base is too large. Make it a 20cm*20cm*20cm cube (0.2 meters). Keep in mind that the unit is always meters. 
and make sure the based on positioned on the "ground"